doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1819,23 +1845,83 @@ It is expected that the restore command is run by the xroad user.
+    backup-public-key-path=/etc/xroad/backupkeys
+
+Backups are always signed. To turn on backup encryption add or change the `backup-encrypted` key to true. If needed also 
+change the folder where additional backup encryption keys are stored. Default is `/etc/xroad/backupkeys`.
+
+Backup encryption keys folder must contain only GPG public key files. In addition to security servers public key backup
+archives are encrypted with all the keys in this folder. It is recommended to use at least one additional public key,
+otherwise the backups will be unusable in case security servers private key is lost. It is up to security servers administrator
+to check that private keys used are sufficiently strong, there are no automatic checks.
+
+Additional keys for backup encryption should be generated and stored outside security server in a secure environment.
+After gpg keypair has been generated, public key can be exported to a file (backupadmin@organisation.eu is the name of the
+key being exported) using this command:
+
+    gpg --output backupadmin.publickey --armor --export backupadmin@organisation.eu
+
+Resulting file `backupadmin.publickey` should be moved to security server and copied to backup encryption keys folder.
+Transfer of the public key does not have to be secure but administrator has to make sure that file is not changed during
+transfer, for example validating file hash.
+
+Private keys corresponding to additional backup encryption public keys must be handled safely and kept in secret. Any of
+them can be used to decrypt backups and thus mount attacks on the security servers.
+
+### 13.5 Verifying Backup Archive Consistency
+
+Security server verifies consistency of backup archives automatically. It is also possible to check archives externally.
+For checking consistency security servers public key is needed. When backups are encrypted then a private key for decrypting
+archive is also needed (GPG uses "sign then encrypt" scheme, so it is not possible to verify encrypted archives without
+decrypting them).
+
+Automatic backup verification is only possible when original security server keypair is available. Should keypair on
+the security server be lost for whatever reason, automatic verification is no longer possible. It is recommended to
+export backup encryption public key and import it into separate secure environment. If backups are encrypted security
+server public key should be imported to keyrings holding additional encryption keys, so that backups can be decrypted
+and verified in these separate environemnts.
+
+To export security servers backup encryption public key use the following command:
+
+    gpg --homedir /etc/xroad/gpghome --armor --output server-public-key.gpg --export AA/GOV/TS1OWNER/TS1
+
+where `AA/GOV/TS1OWNER/TS1` is the security server id. 
 
+Resulting file (server-public-key.gpg) should then be exported from security server and imported to GPG keystore used
+for backup archive consistency checking.
```

Comment:
I would change this part like this:

Security server verifies consistency of backup archives automatically. Also, it is possible to verify the consistency of the archives externally. For verifying the consistency externally, security server's public key is needed. When backups are encrypted, then a private key for decrypting archive is also needed. GPG uses "sign then encrypt" scheme, so it is not possible to verify encrypted archives without decrypting them.

Automatic backup verification is only possible when original security server keypair is available. Should keypair on the security server be lost for whatever reason, automatic verification is no longer possible. Therefore, it is recommended to export backup encryption public key and import it into separate secure environment. If backups are encrypted, security server public key should be imported to keyrings holding additional encryption keys, so that backups can be decrypted and verified in these separate environments.

To export security server's backup encryption public key use the following command:

    gpg --homedir /etc/xroad/gpghome --armor --output server-public-key.gpg --export <instanceIdentifier>/<memberClass>/<memberCode>/<serverCode>

where `<instanceIdentifier>/<memberClass>/<memberCode>/<serverCode>` is the security server id, for example, `AA/GOV/TS1OWNER/TS1`. 

Resulting file (`server-public-key.gpg`) should then be exported from security server and imported to GPG keystore used for backup archive consistency verification.